Meadowlark-js/backends/meadowlark-mongodb-backend/src/repository/OwnershipSecurity.ts

```diff
@@ -35,30 +38,45 @@ export async function rejectByOwnershipSecurity(
   }
 
   const mongoCollection: Collection<MeadowlarkDocument> = getDocumentCollection(client);
-  let id = frontendRequest.middleware.pathComponents.resourceId;
+  const { documentUuid } = frontendRequest.middleware.pathComponents;
 
-  if (id == null) id = extractIdIfUpsert(frontendRequest);
-
-  if (id == null) {
-    Logger.error(`${functionName} - no id to secure against`, frontendRequest.traceId);
-    return 'NOT_APPLICABLE';
-  }
+  let result: WithId<MeadowlarkDocument> | null = null;
 
   try {
-    const result: WithId<MeadowlarkDocument> | null = await mongoCollection.findOne(
-      { _id: id },
-      { projection: { createdBy: 1, _id: 0 } },
-    );
-    if (result === null) {
-      Logger.debug(`${functionName} - document not found for id ${id}`, frontendRequest.traceId);
-      return 'NOT_APPLICABLE';
+    if (documentUuid != null) {
```

Comment:
Something is not working correctly in the ownership security.

Try manually testing with file `local.33b.http`.  Create an `EducationContents` records (line 119). Try the GET ALL that follows it. In my case, this _does_ include the new record. Note that it is an OpenSearch query. Then try the GET BY ID that follows that (line 136). I saw it succeed once, and then fail repeatedly. The PUT request also failed. Interestingly,  rerunning the POST request works.

```
2023-02-28 08:21:30.802 info 1564839f-36f2-474f-be08-87e80939f564 mongodb.repository.OwnershipSecurity.rejectByOwnershipSecurity /v3.3b/ed-fi/EducationContents/1c6674ba-64d2-4068-beb9-b21fb4ff124a
2023-02-28 08:21:30.804 debug 1564839f-36f2-474f-be08-87e80939f564 mongodb.repository.OwnershipSecurity.rejectByOwnershipSecurity - document not found for documentUuid 1c6674ba-64d2-4068-beb9-b21fb4ff124a
```

Reply:
This problem is related to the POST returning different uuids each time. Now it is fixed